newIDE/app/src/MainFrame/Preferences/PreferencesContext.js

```diff
@@ -109,6 +110,7 @@ export type PreferencesValues = {|
   useGDJSDevelopmentWatcher: boolean,
   eventsSheetUseAssignmentOperators: boolean,
   showEffectParameterNames: boolean,
+  lastUsedPath: { [string]: { [ResourceKind]: string } },
```

Comment:
Can we name this: projectLastUsedPaths

To hint that the first key is "a project".

Reply:
Oh! that's a good enhancement :)